.config/starte2e.sh

```diff
@@ -10,6 +10,9 @@ export PATH="${PATH}:/usr/bin"
 touch tests/environment/zitadel/service-user.json
 chmod a+rw tests/environment/zitadel/service-user.json
 
+# Generate certificates
+tests/environment/certs/generate-certs.sh
```

Comment:
We should consider doing this in the docker container to avoid the openssl dependency

Reply:
What using an openssl container just for that? It very lightweight.

Reply:
Yeah, that seems reasonable